### ISSUE TITLE
New releases do not bump up version information

### ISSUE THREAD

**Issue by tomkv** [flathub/org.remmina.Remmina#25]
New releases do not bump up version information:
```
$ flatpak info org.remmina.Remmina

Remmina - Remote Desktop Client

          ID: org.remmina.Remmina
         Ref: app/org.remmina.Remmina/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.4.3
     License: GPL-2.0+
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 14,4 MB
     Runtime: org.gnome.Platform/x86_64/3.36
         Sdk: org.gnome.Sdk/x86_64/3.36

      Commit: 8241b7de2b024ddcab36a245470c9b3e5a5910605fafd8f97e8b1b5797d57e21
      Parent: 654391089105a06f53ee1f118164261723f90843fe233bf3be5c74cd19c961a2
     Subject: Upstream release v1.4.5 (c5c192ee)
        Date: 2020-05-24 21:57:38 +0000
```

I.e. **Version** says `1.4.3` and **Subject** says `Upstream release v1.4.5 (c5c192ee)`.

**Comment by antenore** [comment]
Thanks for this, I always forget about the desktop meta data file!!

**Comment by antenore** [comment]
Build in progress.

**Comment by antenore** [comment]
Done

```sh
$ flatpak info org.remmina.Remmina

Remmina - Remote Desktop Client

          ID: org.remmina.Remmina
         Ref: app/org.remmina.Remmina/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.4.5
     License: GPL-2.0+
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 14.4 MB
     Runtime: org.gnome.Platform/x86_64/3.36
         Sdk: org.gnome.Sdk/x86_64/3.36

      Commit: e56468f2cabc0eec4fa27ef30b7917b03a93a1100bd8a2b6324677ef200d751d
      Parent: 8241b7de2b024ddcab36a245470c9b3e5a5910605fafd8f97e8b1b5797d57e21
     Subject: Adding patch to fix version number in the org.remmina.Remmina.appdata.xml file, closes #25 (0b09f498)
        Date: 2020-05-25 14:00:18 +0000
```

**Comment by tomkv** [comment]
Thanks. :thumbsup:
